### ISSUE TITLE
Visual Editor dataframes are squeezed

### ISSUE THREAD

**Issue by noahlove** [rstudio/rstudio#9034]
### System details

    RStudio Edition : Server
    RStudio Version :  1.4.1106
    OS Version      :  Runs on Ubuntu 20.04, accessed from windows in chrome
    R Version       :  3.6.3

### Steps to reproduce the problem

`head(cars)`

### Describe the problem in detail
The new visual editor constrains the columns of the dataframe really tightly. Cuts off row numbers in some cases and most words are abbreviated (sp...)

### Describe the behavior you expected
Spread out more like non visual editor r studio ide. Makes the amazing new visual editor frustrating to use. 

![r-studio bug](https://user-images.githubusercontent.com/57121482/109262819-0fa31a80-77b7-11eb-8029-f05a2e7029c9.JPG)

- [ x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [ x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x ] If I am reporting a RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x ] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by noahlove** [comment]
![alternative](https://user-images.githubusercontent.com/57121482/109263041-77596580-77b7-11eb-8ae1-93b8df8ad963.JPG)

Although I'm sure everyone knows how it looks, this is what it looks like when turning off visual editor and returning to the "normal" interface

**Comment by jmcphers** [comment]
Thanks for writing this up! I've seen this myself.

**Comment by noahlove** [comment]
@jmcphers Of course! It was actually my first ever issue so that was exciting as well!

**Comment by jmcphers** [comment]
@noahlove Are you still seeing this with the latest version? I'm not able to reproduce, by either:

1. Creating a new chunk in visual mode and running it
2. Creating a new chunk in source mode and running it, then switching to visual mode
3. Clearing the chunk output, running it again

**Comment by ronblum** [comment]
I can still reproduce this in
- RStudio Desktop Ghost Orchid 2021.09.1+372 on MacOS 12.1
- RStudio Desktop Prairie Trillium 2022.02.0-daily+335 on MacOS 12.1

If I close RStudio and reopen it, the columns are displayed correctly.

**Comment by bleds22e** [comment]
Has there been movement on this? It happens both in RStudio desktop (2024.12.0+467 on MacOS 15.3.1) and in Posit Cloud.

I LOVE using visual mode to teach (I'm usually introducing undergrads to computer programming for the first time), but having the column names condense is definitely frustrating.
